.travis.yml

```diff
@@ -88,6 +88,8 @@ install:
   - travis_retry sudo luarocks install busted
   # Install luacheck for "make check-qa".
   - if [ "$DO_CHECKQA" = 1 ]; then travis_retry sudo luarocks install luacheck; fi
+  # Install depgraph for "make check-requires".
+  - travis_retry sudo luarocks install depgraph
```

Comment:
Rebased on master and added a fixup commit to address the above (moving `check-qa` to another travis target is outside the scope of this PR)